RecoBTag/CTagging/python/RecoCTagging_cff.py

```diff
@@ -5,10 +5,15 @@
 # but IP and soft-lepton taginfos from btagging sequence can be recycled
 pfCTagging = cms.Sequence(
     ( inclusiveCandidateVertexingCvsL *
-      pfInclusiveSecondaryVertexFinderCvsLTagInfos
+      pfInclusiveSecondaryVertexFinderCvsLTagInfos*
+      pfInclusiveSecondaryVertexFinderNegativeCvsLTagInfos 
     )
 
     # CSV + soft-lepton variables combined (ctagger optimized for c vs dusg)
     * pfCombinedCvsLJetTags
+    * pfNegativeCombinedCvsLJetTags
+    * pfPositiveCombinedCvsLJetTags
     * pfCombinedCvsBJetTags
+    * pfNegativeCombinedCvsBJetTags
+    * pfPositiveCombinedCvsBJetTags
```

Comment:
same comment applies here. In this way you are making the negative taggers a default in the reco sequence, while we want only the option to run them if we really want.